### ISSUE TITLE
Game not found (Heroic)

### ISSUE THREAD

**Issue by auron95** [PhilipK/BoilR#109]
I installed the game "Brothers - A Tale of Two Sons" from the Epic Game Store (via Heroic) but BoilR cannot find it. Other games are found just fine (tried with CrisTales, always from Epic). Wild guess: maybe some problem with spaces or the dash in the name of the game?
Let me know if you need further info about it.

**Comment by PhilipK** [comment]
Are the games installed on the SD card?

**Comment by auron95** [comment]
No, I have no SD card inserted, everything is on the internal storage.

**Comment by mkmatis** [comment]
I have similar issue with Hyper Light Drifter. Other games (Hades, Into the Breach and Rogue Legacy) are detected correctly. All games are installed on SD card in the same parent directory.

**Comment by PhilipK** [comment]
Can someone try to upload thier installed.json file? Then maybe I can debug what is wrong.

The location is :

/home/deck/.var/app/com.heroicgameslauncher.hgl/config/legendary/installed.json

**Comment by PhilipK** [comment]
Alright, version 1.3.14 coming up, it should fix this problem, let me know if it is still a problem for you and I will reopen this.
